web/src/tracking/eventLogger.ts

```diff
@@ -26,14 +26,14 @@ export class EventLogger implements TelemetryService {
      * Log a pageview.
      * Page titles should be specific and human-readable in pascal case, e.g. "SearchResults" or "Blob" or "NewOrg"
      */
-    public logViewEvent(pageTitle: string, logAsActiveUser = true): void {
+    public logViewEvent(pageTitle: string, eventProperties?: any, logAsActiveUser = true): void {
         if (window.context?.userAgentIsBot || !pageTitle) {
             return
         }
         pageTitle = `View${pageTitle}`
 
         const props = pageViewQueryParameters(window.location.href)
-        serverAdmin.trackPageView(pageTitle, logAsActiveUser)
+        serverAdmin.trackPageView(pageTitle, true, eventProperties)
```

Comment:
```suggestion
        serverAdmin.trackPageView(pageTitle, logAsActiveUser, eventProperties)
```

There are only a few pages where this isn't `true` (you edited those files above, actually), but important to retain that value when it is set.

Reply:
Thanks for catching this!